data/json/recipes/food/brewing.json

```diff
@@ -151,113 +144,109 @@
     "subcategory": "CSC_FOOD_BREW",
     "skill_used": "cooking",
     "difficulty": 2,
-    "time": "40 m",
-    "batch_time_factors": [ 50, 4 ],
+    "time": "30 m",
+    "batch_time_factors": [ 95, 4 ],
     "book_learn": [ [ "brewing_cookbook", 1 ], [ "family_cookbook", 1 ] ],
-    "qualities": [ { "id": "COOK", "level": 2 } ],
+    "qualities": [ { "id": "BOIL", "level": 2 } ],
     "tools": [ [ [ "surface_heat", 30, "LIST" ] ] ],
     "components": [ [ [ "water", 15 ], [ "water_clean", 15 ] ], [ [ "pumpkin", 12 ] ], [ [ "sugar", 50 ] ], [ [ "yeast", 2 ] ] ]
   },
   {
     "type": "recipe",
-    "activity_level": "LIGHT_EXERCISE",
+    "activity_level": "NO_EXERCISE",
     "result": "brew_fruit_wine",
     "result_mult": 3,
     "category": "CC_FOOD",
     "subcategory": "CSC_FOOD_BREW",
     "skill_used": "cooking",
     "difficulty": 3,
-    "time": "40 m",
-    "batch_time_factors": [ 50, 4 ],
+    "time": "30 s",
+    "batch_time_factors": [ 95, 4 ],
     "book_learn": [ [ "brewing_cookbook", 2 ], [ "winemaking_beginner", 2 ] ],
-    "qualities": [ { "id": "COOK", "level": 2 } ],
-    "tools": [ [ [ "surface_heat", 6, "LIST" ] ] ],
-    "components": [ [ [ "water", 3 ], [ "water_clean", 3 ] ], [ [ "sweet_fruit", 2, "LIST" ], [ "juice", 1 ] ], [ [ "yeast", 1 ] ] ]
+    "qualities": [ { "id": "CONTAIN", "level": 1 } ],
+    "components": [ [ [ "juice", 1 ] ], [ [ "yeast", 1 ] ] ]
   },
   {
     "type": "recipe",
-    "activity_level": "LIGHT_EXERCISE",
+    "activity_level": "NO_EXERCISE",
     "result": "brew_hb_beer",
-    "result_mult": 15,
+    "result_mult": 10,
     "category": "CC_FOOD",
     "subcategory": "CSC_FOOD_BREW",
     "skill_used": "cooking",
     "difficulty": 4,
-    "time": "80 m",
-    "batch_time_factors": [ 50, 4 ],
+    "time": "30 m",
```

Comment:
60 minutes would be better here. 30 minutes to mash the grains followed by a 30 minute boil to isomerize the hop oils.

Reply:
Seems reasonable, what about the batch time? Boiling part should be a reasonable batch time although mashing would be labour intensive and more involved for a larger quantity.